modules/_labs/tabs/react/index.ts

```diff
@@ -1,7 +1,7 @@
-import Tabs from './lib/Tabs';
-import TabList from './lib/TabList';
-import Tab from './lib/Tab';
-import TabPanel from './lib/TabPanel';
+import {Tabs} from './lib/Tabs';
+import {TabList} from './lib/TabList';
+import {Tab} from './lib/Tab';
+import {TabPanel} from './lib/TabPanel';
```

Comment:
We haven't talked about this one, but there is so much extra boilerplate in default exports. How are they serving us now? Components like 'Button' have to decide which button export is "default" and we have to remap default exports of almost every file that uses them. Why not always use named exports?